answerking_app/views/category_views.py

```diff
@@ -23,7 +23,6 @@ class CategoryListView(
     queryset: QuerySet = Category.objects.all()
     serializer_class: CategorySerializer = CategorySerializer
 
-    @csrf_exempt
```

Comment:
Have we got rid of all the @csrf_exempt decorators now?

Reply:
Yes as we're running in a dev environment, which gets rid of `django.middleware.csrf.CsrfViewMiddleware`, meaning we don't need to explicitly add decorators